addons/base_geolocalize/models/res_partner.py

```diff
@@ -12,10 +12,10 @@ class ResPartner(models.Model):
     def _geo_localize(self, street='', zip='', city='', state='', country=''):
         geo_obj = self.env['base.geocoder']
         search = geo_obj.geo_query_address(street=street, zip=zip, city=city, state=state, country=country)
-        result = geo_obj.geo_find(search)
+        result = geo_obj.geo_find(search, **dict(force_country=country))
```

Comment:
what's wrong with
```suggestion
        result = geo_obj.geo_find(search, force_country=country)
```
?

Reply:
why ? 

`def geo_find(self, addr, **kw):`

take a dict as param

Reply:
creating a dict just for unpacking it is useless.

Reply:
thx for good tips, just removed all both occurrences ;)